frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -193,6 +177,23 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
                 groups: [],
             })
         },
+        loadMorePeople: async ({}, breakpoint) => {
+            if (values.people) {
+                const { people: currPeople, count, action, label, day, breakdown_value, next } = values.people
+                const morePeopleResults = await api.get(next)
+                breakpoint()
+                const morePeople = {
+                    people: [...currPeople, ...morePeopleResults.results[0]?.people],
+                    count: count + morePeopleResults.results[0]?.count,
+                    action,
+                    label,
+                    day,
+                    breakdown_value,
+                    next: morePeopleResults.next,
+                }
+                actions.setLoadMorePeople(morePeople)
+            }
+        },
```

Comment:
I think this could still be done in the loader. It adds nice `Success` and `Failure` actions we can hook into if we need to, and just consolidates requests. Is there an option to make the difference in React, so that it doesn't switch to a full skeleton when people are loading, but only if it's the first load? (e.g. when people === null)